crates/wasm/publish/pnpm-lock.yaml

```diff
@@ -0,0 +1,477 @@
+lockfileVersion: '6.0'
+
```

Comment:
I think this was staged by accident?

Reply:
Yeah, I have no idea why that was generated